Mathlib/Algebra/Category/ModuleCat/Sheaf/Generators.lean

```diff
@@ -73,4 +88,39 @@ def equivOfIso (e : M ≅ N) :
 
 end GeneratingSections
 
+variable [∀ (X : C), HasWeakSheafify (J.over X) AddCommGrp.{u}]
```

Comment:
Maths question: how close is `∀ (X : C), HasWeakSheafify (J.over X) AddCommGrp.{u}` to just `HasWeakSheafify J AddCommGrp.{u}`? Do you know examples where it works for J but fails for some J.over X? This is independent of the merging process of course, I was just interested.